src/booking/containers/booking-check/booking-check.component.ts

```diff
@@ -51,6 +51,17 @@ export class BookingCheckComponent implements OnInit {
       beginDate: this.bookingProcess.startDate,
       endDate: this.bookingProcess.endDate
     };
+    const givenDate = payload.beginDate;
```

Comment:
@connorpgpmcelroy  you should not be doing the conversion yourself. Rather something like this https://stackoverflow.com/questions/6525538/convert-utc-date-time-to-local-date-time